components/eamxx/scripts/machines_specs.py

```diff
@@ -146,6 +165,23 @@ def setup(cls):
         cls.num_run_res = 4 # four gpus
         cls.gpu_arch = "cuda"
 
+###############################################################################
+class Polaris(CrayMachine):
+###############################################################################
+    concrete = True
+    @classmethod
+    def setup(cls):
+        super().setup_base("polaris")
+
+        compiler = "gnugpu"
+
+        cls.env_setup = [f"eval $({CIMEROOT}/CIME/Tools/get_case_env -c SMS.ne4pg2_ne4pg2.F2010-SCREAMv1.{cls.name}_{compiler})"]
+        
+        # TODO: check in with az if this is correct
+        cls.batch = "qsub -q debug_scaling -l walltime=01:00:00 -A E3SM_RRM"
+        cls.gpu_arch = "cuda"
```

Comment:
Not that it matters, but you should prob not set this to "cuda". For the record, this setting is queried to determine if running compute-sanitizer or valgrind, mostly. Since we are not (yet) running memory checking on polaris, this is not that important. Also, there is no support for anything other than valgrind or compute-sanitizer (for the memcheck build), so...